src/main.py

```diff
@@ -130,22 +122,14 @@ def order_subscription():
 @login_required
 def order_packets():
     form = OrderPacketsForm()
-    descriptions = [(manifest.PACKETS[name]["name"], manifest.PACKETS[name]["description"]) for name in
-                    manifest.PACKETS]
+    descriptions = [(manifest.PACKETS[name]["name"], manifest.PACKETS[name]["description"]) for name in manifest.PACKETS]
     descriptions = dict(descriptions)
 
     if form.validate_on_submit():
-        user = db.get_client_by_email(form.email.data)
-        user.bundles.append(BundleInfo(email=form.email.data,
-                                       name=form.packets.data,
-                                       date_from=datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
-                                       date_to=(datetime.datetime.now() + datetime.timedelta(days=manifest.PACKETS[form.packets.data]['duration'])).strftime("%Y-%m-%d %H:%M:%S")
-                                       ))
-        db.serialize(user)
+        # place for processing from data
```

Comment:
why removed ordering packets?

Reply:
I haven't noticed it during fixing merge conflicts but it should be alright now

Reply:
problem solved